Makefile

```diff
@@ -1,11 +1,10 @@
-V=20220223
+V=20241216
 
 PREFIX = /usr/local
 
 install:
 	install -dm755 $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
-	# install -m0644 asahi-alarm{.gpg,-trusted,-revoked} $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
-	install -m0644 asahi-alarm{.gpg,-trusted} $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
+	install -m0644 asahi-alarm{.gpg,-trusted,-revoked} $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
```

Comment:
Why removed the `-revoked`?

Reply:
because the file doesn't exist (we don't have revoked keys). We could consider revoking marcan's key

Reply:
But does installation fail when there is no file? I think it's ok to just have an empty file as well

Reply:
I pushed an empty revoked file now.
(If you check current `/usr/share/pacman/keyrings/asahilinux-revoked` is also empty)